### PR TITLE
[FEATURE] Add UI5_LOG_LVL environment variable

### PR DESCRIPTION
**Pull request by RandomByte**
To allow easy log level changes in scenarious where only single UI5 Tooling modules are being used

### CODE REVIEW

File: lib/logger.js [hunk 1]
```diff
@@ -1,5 +1,17 @@
 const npmlog = require("npmlog");
-npmlog.level = "info"; // Our default
+
+if (process.env.UI5_LOG_LVL) {
+	const levels = ["silly", "verbose", "info", "warn", "error"];
+	const logLvl = process.env.UI5_LOG_LVL;
+	if (levels.indexOf(logLvl) === -1) {
```
Comment (matz3) [inline lib/logger.js]:
```suggestion
	if (!levels.includes(logLvl)) {
```

File: lib/logger.js [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+		throw new Error(`UI5 Logger: Environment variable UI5_LOG_LVL is set to an unkown log level "${logLvl}". ` +
+			`Valid levels are ${levels.toString()}`);
```
Comment (matz3) [inline lib/logger.js]: Maybe add a space to the list of levels?
```suggestion
			`Valid levels are ${levels.join(", ")}`);
```

File: lib/logger.js (hunk 1 above)
Comment (RandomByte) [reply lib/logger.js]: Done

File: lib/logger.js (hunk 2 above)
Comment (RandomByte) [reply lib/logger.js]: Done

**Pull request merged**